crates/ef-testing/src/models/case.rs

```diff
@@ -156,7 +156,6 @@ impl BlockchainTestCase {
             ));
         }
         let gas_price = gas_price | effective_gas_price;
-        let transaction_cost = gas_price * expected_gas_used;
```

Comment:
we don't need this anymore ?

Reply:
No this isn't used anymore, was removed in a PR by eni